src/longopt/grdinterpolate_inc.h

```diff
@@ -37,7 +37,7 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = { /* Local options for this m
 	{ 0, 'S', "pointseries",
 	          "",                      "",
 	          "h",                     "header" },
-	{ 0, 'T', "timeinc",
+	{ 0, 'T', "timeinc|inc|range",
```

Comment:
``timeinc`` is associated with time but -T is for any type of increments.

Reply:
All of the initial entries in the above alias lists were the pre-existing ones approved earlier. The aliases I added were either pre-approved by Paul or added by me after looking at Julia and trying to add compatibility.

Personally I do not really care about any of these that much. The rest of you can discuss and I will make whatever changes Paul directs me to make.